infrastructure/models.py

```diff
@@ -50,22 +66,40 @@ def __str__(self):
 
 
 class ProjectStatus:
+    NULL = 0
     ANNOUNCED = 1
     PREPATORY = 2
     STARTED = 3
     UNDER_CONSTRUCTION = 4
     COMPLETED = 5
     CANCELLED = 6
+    ACTIVE = 7
+    PARTIALLY_ACTIVE = 8
+    INACTIVE = 9
 
     STATUSES = (
```

Comment:
Based on the sheets, we will also need a DECOMMISSIONED and a SUSPENDED status to be added to the `ProjectStatus` options